### PR TITLE
Login with Version Control announcement

### PR DESCRIPTION
**Pull request by davidfischer**
This is a pretty short blog but it should make for a good announcement. I'm aiming for a week from today which is between WTD and PyCon.

This feature was added here: https://github.com/rtfd/readthedocs.org/pull/4022.

I considered putting an image (either static or an animated gif) but I don't think it is necessary. An image of the login screen has a lot of stuff that isn't "log in with ..." related.

### CODE REVIEW

File: version-control-login.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
+.. post:: May 9, 2018
+   :tags: feature, new-user, versioncontrol
+   :author: David
+   :location: SAN
+
+
+Version Control Log in
```
Comment (humitos) [inline version-control-login.rst]: I'd say that the title should reflect the "social" part of the version control. I mean, you don't log in with git, but with github.

> Social Version Control Log in

Maybe?

File: version-control-login.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+
+For new Read the Docs users
+---------------------------
+
+For new users, the sign up process is significantly streamlined.
+There's no new password to remember and when you're ready to start building your docs,
+Read the Docs will be ready with a list of your repositories to get started.
+
+Click the "sign up" button on the Read the Docs home page and then click
```
Comment (humitos) [inline version-control-login.rst]: "Sign up"

File: version-control-login.rst [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+the button for your favorite version control host.
```
Comment (humitos) [inline version-control-login.rst]: I think it's more accurate to use the same wording you used in the first paragraph:

"version control hosting service."

**Comment by davidfischer** [comment]
> My only feedback with this is that we should make sure this is out on the .com as well.

I haven't contributed on the .com but I can take a look.

**Comment by agjohnson** [comment]
I opened an issue for @humitos to look at, I'd probably just vote for delaying the blog post until this is out and tested.

**Pull request merged**